src/jit/compiler.cpp

```diff
@@ -2560,6 +2560,79 @@ void Compiler::compSetProcessor()
 #endif // DEBUG
 
 #endif // _TARGET_X86_
+
+// Instruction set flags fo// Instruction set flags for Intel hardware intrinsics
+#ifdef _TARGET_XARCH_
+    opts.compSupportsAES       = false;
```

Comment:
Perhaps a single "flags" variable should be used instead of all these variables?

Reply:
Agree, will change to use a 64-bit flag variable.